### PR TITLE
Bahc3 77 semantic markup

### PR DESCRIPTION
**Pull request by sidvlad**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,110 +2,424 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <!-- <link rel="stylesheet" href="style.css"> --> <!-- uncomment styles --> 
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Binabox NFT</title>
```
Comment (andacenko) [inline index.html]: Just Binabox

File: index.html [hunk 2]
```diff
@@ -2,110 +2,424 @@
...
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+    <header class="header">
+            <a href="#" class="logo">
+                <img src="images/logo.png" alt="Blog Post" class="logo__image">
```
Comment (andacenko) [inline index.html]: 1. .svg
2. logo-white
3. width="" height=""
4. alt="Binabox"
5. check all images and add width="" height="" alt=""

File: index.html [hunk 3]
```diff
@@ -2,110 +2,424 @@
...
-          <div class="folder">
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+    <header class="header">
+            <a href="#" class="logo">
```
Comment (andacenko) [inline index.html]: For home.html change `<a>` to `<h1>`

File: index.html [hunk 4]
```diff
@@ -2,110 +2,424 @@
...
+                <img src="images/logo.png" alt="Blog Post" class="logo__image">
+                <div class="logo__text">binabox</div>
```
Comment (andacenko) [inline index.html]: Delete this block

File: index.html [hunk 5]
```diff
@@ -2,110 +2,424 @@
...
+                </ul>
+            </nav>
+
+            <div class="block__btns">
+                <form class="search__form" action="/search" method="GET">
+                    <input class="search__form--input" type="text" placeholder="Search" aria-label="Search">
+                    <svg id="search" class="search__form--icon" viewBox="0 0 24 24">
+                        <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/>
+                        <path d="M0 0h24v24H0z" fill="none"/>
+                    </svg>
```
Comment (andacenko) [inline index.html]: Wrap svg into `<button type="submit">`

File: index.html [hunk 6]
```diff
@@ -2,110 +2,424 @@
...
+                    </li>
+                    <li class="header__menu--item">
+                        <a href="#" class="header__menu--link">Contact Us</a>
+                    </li>
+                </ul>
+            </nav>
+
+            <div class="block__btns">
+                <form class="search__form" action="/search" method="GET">
+                    <input class="search__form--input" type="text" placeholder="Search" aria-label="Search">
```
Comment (andacenko) [inline index.html]: Change order button then input

File: index.html [hunk 7]
```diff
@@ -2,110 +2,424 @@
...
+                    <svg id="search" class="search__form--icon" viewBox="0 0 24 24">
+                        <path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/>
+                        <path d="M0 0h24v24H0z" fill="none"/>
+                    </svg>
+                </form >
+                <div class="control__divider"></div>
```
Comment (andacenko) [inline index.html]: Don't use empty blocks. Move all decorative elements into css. Check all places.

File: index.html [hunk 8]
```diff
@@ -2,110 +2,424 @@
...
+    </header>
 
-            <div class="body">
-              <div class="profile-icon-box">
-                <p id="typeText"></p>
-                <div class="profile-icon">
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon.png" /> <!-- Photo -->
+    <section class="intro">
+        <div class="intro__wrapper">
+            <p class="intro__title">With <span>BINABOX</span> Explore NFT Collection</p>
```
Comment (andacenko) [inline index.html]: Use `<strong>` instead `<p>`

File: index.html [hunk 9]
```diff
@@ -2,110 +2,424 @@
...
+                    <p class="intro__text">Products unlocked</p>
                 </div>
-              </div>
+            </div>
+            <div class="block__btns">
+                <a href="#" class="btn btn__color">Connect Wallet</a>
+                <a href="#" class="btn">Whitelist Now</a>
+            </div>
+            <div class="intro__bg--main"><img src="images/bg.png" alt="background intro"></div>
+            <div class="intro__bg--second"><img src="images/bg2.png" alt="background intro"></div>
```
Comment (andacenko) [inline index.html]: Move second img to the css

File: index.html [hunk 10]
```diff
@@ -2,110 +2,424 @@
...
-              <div class="name">
-                <p>AYAKO YOKOYA</p> <!-- Name -->
-              </div>
-              <div class="aboutme">
-                <p>Creative Developer <span>(I wanna be...)</span></p> <!-- Profession -->
-              </div>
+    <main class="main">
+        <section class="catalog">
+            <div class="catalog__navigation">
+                <h3 class="catalog__title">Catalog</h3>
```
Comment (andacenko) [inline index.html]: Use `<h2>`

File: index.html [hunk 11]
```diff
@@ -2,110 +2,424 @@
...
+                <hr class="divider">
+                <div class="block__btns">
+                    <a href="#" class="btn">See all</a>
+                    <div class="control__divider"></div>
+                    <a href="#" class="btn"></a>
+                    <a href="#" class="btn btn__color"></a>
```
Comment (andacenko) [inline index.html]:
```
<a href="#"><svg><span class="hidden">previous button</span></a>
<a href="#"><svg><span class="hidden">next button</span></a>
```

File: index.html [hunk 12]
```diff
@@ -2,110 +2,424 @@
...
+            <h2 class="section__title">Our Blog</h2>
+            <div class="blog__wrapper">
+                <article class="blog__post">
+                    <div class="blog__post--image">
+                        <img src="images/blog-image.jpg" alt="Blog Post">
+                        <span class="label label__sold">Sold out</span>
+                    </div>
+                    <div class="blog__post-info">
+                        <p class="blog__post-info--name">Tony Nguyen</p>
+                        <p class="blog__post-info--date">20 Jun 2022</p>
```
Comment (andacenko) [inline index.html]: `<time datetime="gg/mm/dd"></time>`

File: index.html [hunk 13]
```diff
@@ -2,110 +2,424 @@
...
+                    </div>
+                    <h3 class="blog__post--title">Sonic Live</h3>
+                    <p class="blog__post--description">One September 22, 2021 A volutpat nulla eros, nulla ultrices nec nulla imperdiet...</p>
+                    <a href="#" class="btn">Read more > </a>
```
Comment (andacenko) [inline index.html]: `>` it's icon

File: index.html [hunk 14]
```diff
@@ -2,110 +2,424 @@
...
+                <a href="#" class="btn">Read more</a>
+            </div>
+        </section>
+
+        <section class="about">
+            <div class="about__wrapper">
+                <h2 class="section__title">About Us</h2>
+                <p class="about__subtitle">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat cupidatat non proident, sunt in culpa qui officia dese runt mollit anim id est laborum velit esse cillum dolore eu fugiat nulla pariatu epteur sint occaecat</p>
+                
+                <ul class="about__features">
```
Comment (andacenko) [inline index.html]: `<ol>`

File: index.html [hunk 15]
```diff
@@ -2,110 +2,424 @@
...
+                    <li class="about__feature">
+                        <h3 class="about__feature--title">High Quality</h3>
+                        <p class="about__feature--description">Lorem ipsum dolor sit amet.</p>
+                        <p class="about__feature--number">01</p>
```
Comment (andacenko) [inline index.html]: numbers move to css ::after

File: index.html [hunk 16]
```diff
@@ -2,110 +2,424 @@
...
+                    </li>
```
Comment (andacenko) [inline index.html:236]: wrap all li content into `<a>` we have hover state in the design

File: index.html [hunk 17]
```diff
@@ -2,110 +2,424 @@
...
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+        <section class="team">
+            <h2 class="section__title">Our Team</h2>
+            <div class="team__members">
+                <article class="team__member">
+                    <img src="images/team.jpg" alt="Team Member" class="team__member--image">
+                    <h3 class="team__member--name">Jason Smith</h3>
+                    <p class="team__member--role">Co-founder</p>
```
Comment (andacenko) [inline index.html]: wrap img h3 p into link

File: index.html [hunk 18]
```diff
@@ -2,110 +2,424 @@
...
+            <div class="logo__wrapper">
+                <a href="#" class="logo">
+                    <img src="images/logo.png" alt="Blog Post" class="logo__image">
+                    <div class="logo__text">binabox</div>
+                </a>
+                <div class="footer__description">We are the best way to check the rarity of NFT collection.</div>
+            </div>
+            <div class="footer__nav">
+                <div class="footer__nav-block">
+                    <div class="footer__nav-title">Subscribe</div>
```
Comment (andacenko) [inline index.html]: `<strong>`

File: index.html [hunk 19]
```diff
@@ -2,110 +2,424 @@
...
+                </div>
             </div>
-          </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
+        <hr class="divider">
+        <div class="footer__secondary">
+            <div class="footer__copyright"><span>Binabox</span> 2022 - ALL rights reserved</div>
+            <div class="footer__social">
```
Comment (andacenko) [inline index.html:509]: use social links structure through all project

File: index.html (hunk 1 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 6 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 7 above)
Comment (sidvlad) [reply index.html]: done & understand

File: index.html (hunk 9 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 11 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 12 above)
Comment (sidvlad) [reply index.html]: done

File: index.html (hunk 19 above)
Comment (sidvlad) [reply index.html:509]: done

**Review by sidvlad — COMMENTED** [review]
i`ve done all commits

**Pull request merged**